runtime/onert/core/include/ir/Index.h

```diff
@@ -64,7 +70,12 @@ inline std::ostream &operator<<(std::ostream &o, const IOIndex &i)
 
 inline std::ostream &operator<<(std::ostream &o, const SubgraphIndex &i)
 {
-  return _index_print_impl(o, "SUBGRAPH", i); // $ubgraph
+  return _index_print_impl(o, "SUBGRAPH", i);
```

Comment:
Not related to this PR, however, I would like to fix this typo in comment in this PR if reviewers are okay.